.github/dependabot.yml

```
@@ -95,3 +100,4 @@ updates:
     open-pull-requests-limit: 3
     commit-message:
       prefix: "ci(deps)"
+
```

YAMLlint (1.37.1)

[error] too many blank lines (1 > 0)

(empty-lines)